src/gui/folder.cpp

```diff
@@ -1299,9 +1310,9 @@ void FolderDefinition::save(QSettings &settings, const FolderDefinition &folder)
 
     // Ensure new vfs modes won't be attempted by older clients
     if (folder.virtualFilesMode == Vfs::WindowsCfApi) {
-        settings.setValue(QLatin1String(versionC), 3);
+        settings.setValue(versionC(), 4);
```

Comment:
shouldn't this call `maxSettingsVersion()`?

Reply:
Hmm as you can see two lines later it doesn't always need to be the max version.
But I'll extract the two versions.